.renovaterc

```diff
@@ -27,5 +27,4 @@
   "lockFileMaintenance": {
     "automerge": true
   },
-  "travis": { "enabled": true }
```

Comment:
I made a note to figure out what I need to do to ensure circleCi checks will "just work" with renovate bot prs.